crates/pallet-domains/src/lib.rs

```diff
@@ -1014,19 +1016,12 @@ mod pallet {
         pub fn instantiate_domain(
             origin: OriginFor<T>,
             domain_config: DomainConfig,
-            raw_genesis: Vec<u8>,
         ) -> DispatchResult {
-            let (who, raw_genesis) = if raw_genesis.is_empty() {
-                (ensure_signed(origin)?, None)
-            } else {
-                // TODO: remove once XDM is finished
-                ensure_root(origin)?;
-                (T::SudoId::get(), Some(raw_genesis))
-            };
+            let who = ensure_signed(origin)?;
```

Comment:
Do you mean remove it from the domain runtime's `RuntimeGenesisConfig`?

Reply:
Yeap. We dont need to any Sudo here.

Reply:
The reason we can remove the sudo/raw_genesis from `instantiate_domain` is the new domain instance will use the runtime object's genesis state which already contains the sudo. If the sudo account is removed from the domain runtime's `RuntimeGenesisConfig` then the runtime object's genesis state (and all the domain chain) won't have the sudo at all, it can be an issue when we launch the next gemini network.

Reply:
Sorry, I did not communicated that well. I was suggesting to remove `SudoID` config paramater